.appveyor.yml

```diff
@@ -55,6 +55,7 @@ install:
   - conda config --set always_yes true
   - conda config --set show_channel_urls yes
   - conda config --prepend channels conda-forge
+  - conda update -n base -q conda
```

Comment:
```suggestion
  - conda update -n base -q conda python
```

Reply:
This should only update python to the newest 3.7 afaIk.